tests/README.md

```diff
@@ -1,115 +1,67 @@
+
 ```console
 tests/scripts/minikube.sh update
 tests/scripts/minikube.sh helm
 ```
 
 Stopping the cluster and destroying the Minikube VM can be done with:
-```console
-tests/scripts/minikube.sh clean
-```
 
-#### Kubeadm (recommended for Ubuntu)
-Starting the cluster using `kubeadm` is as simple as running:
 ```console
-tests/scripts/kubeadm.sh up
-```
-
-`kubeadm.sh` starts Kubernetes on the host, so all the images generated by your local build should be available to the cluster without any additional commands.
-You can change the Kubernetes version (default is v1.15.12) with `KUBE_VERSION` environment variable. Here is the example of using Kubernetes v1.19.0.
-
-```console
-export KUBE_VERSION=v1.19.0
-tests/scripts/kubeadm.sh up
+tests/scripts/minikube.sh clean
 ```
 
-Please note that this script may downgrade the following deb packages.
+For faster iteration during development, take a snapshot of the fresh minikube VM and then reset
+the VM each time you need a clean environment.
 
-- kubernetes-cni
-- kubelet
-- kubeadm
+## Install Helm
 
-Stopping the cluster can be done with:
-```console
-tests/scripts/kubeadm.sh clean
-```
-
-#### Alternate Kubernetes Versions
-These two scripts can install any version of Kubernetes you wish based on the `KUBE_VERSION` environment variable.
-To use an alternate version, simply set this variable before running the relevant `up` command from above.
-For example, if you wanted to use `v1.12.5`, you would run `export KUBE_VERSION=v1.12.5` first before running `up`.
-
-### Install Helm
 Use [helm.sh](/tests/scripts/helm.sh) to install Helm and set up Rook charts defined under `_output/charts` (generated by build):
 
 - To install and set up Helm charts for Rook run `tests/scripts/helm.sh up`.
 - To clean up `tests/scripts/helm.sh clean`.
 
-**NOTE:** `*kubeadm.sh`, `minikube.sh` and `helm.sh` scripts depend on some artifacts under the `_output/` directory generated during build time,
-these scripts should be run from project root. e.g., `tests/script/kubeadm.sh up`.
+**NOTE:** These helper scripts depend on some artifacts under the `_output/` directory generated during build time.
+These scripts should be run from project root. e.g., `tests/script/kubeadm.sh up`.
 
 **NOTE**: If Helm is not available in your `PATH`, Helm will be downloaded to a temporary directory (`/tmp/rook-tests-scripts-helm`) and used from that directory.
 
 ## Run Tests
-From the project root do the following:
 
-### 1. Build rook:
-Run `make build`
+From the project root do the following after bringing up your Kubernetes cluster such as minikube:
+
+### 1. Build rook
+
+`make build`
 
 ### 2. Start Kubernetes
-Using one of the following:
 
-- Using Kubeadm
-```
-tests/scripts/kubeadm.sh up
-export KUBECONFIG=~/admin.conf
-tests/scripts/helm.sh up
-```
-- Using minikube
-```
+Copy the local build into minikube:
```

Comment:
```suggestion
Start the minikube cluster and copy the local build into minikube:
```
or maybe some other words as `Copy the local build into minikube` inside `Start Kubernetes` heading seems incomplete.